package.json

```diff
@@ -19,6 +19,8 @@
     "test-visual:ci": "npm run build && npm run build-storybook:web-components && lerna run test-visual:ci --stream",
     "test-visual:update": "lerna run test-visual:update --stream",
     "test-visual:update-ci": "npm run build && lerna run test-visual:update --stream",
+    "test:cypress": "lerna run cypress --scope=@ukic/react --stream",
+    "test:cypress-ci": "lerna run cypress:ci --scope=@ukic/react --stream",
```

Comment:
minor one - could these by `test-cypress` & `test-cypress:ci `to be consistent with test-e2e, test-visual etc